docs/user/quick-start-browser.md

```diff
@@ -0,0 +1,226 @@
+It has to be configured with `app-agent-receiver` integration to expose a http collection
+endpoint and run with `integrations-next` flag enabled.
+
+Below is an example of a basic Grafana Agent config that will expose a collector endpoint
+at [http://host:12345/collect](http://host:12345/collect), and forward collected telemetry to Loki,
+Tempo and Prometheus instances. See [agent app receiver integration documentation](https://github.com/grafana/agent/blob/main/docs/user/configuration/integrations/integrations-next/app-agent-receiver-config.md)
+for more details on available options.
+
+```yaml
+metrics:
+  wal_directory: /tmp/wal
+  global: {}
+  configs:
+    - name: default
+      remote_write:
+        - url: https://prometheus-us-central1.grafana.net/api/prom/push
+          basic_auth:
+            username: xxx
+            password: xxx
+logs:
+  positions_directory: /tmp/loki-pos
+  configs:
+    - name: default
+      scrape_configs: []
+      clients:
+        - url: https://xxx:xxx@logs-prod-us-central1.grafana.net/loki/api/v1/push
+traces:
+  configs:
+    - name: default
+      remote_write:
+        - endpoint: tempo-us-central1.grafana.net:443
+          basic_auth:
+            username: xxx
+            password: xxx
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+            http:
+              cors:
+                allowed_origins:
+                  - http://localhost:1234
+                max_age: 7200
+integrations:
+  app_agent_receiver_configs:
+    - autoscrape:
+        enable: true
+        metrics_instance: "default"
+      api_key: "secret" # optional, if set, client will be required to provide it via x-api-key header
+      instance: "frontend"
+      logs_instance: "default"
+      traces_instance: "default"
+      server:
+        host: 0.0.0.0
+        port: 12345
+        cors_allowed_origins:
+          - "https://my-app.example.com"
+      logs_labels: # labels to add to loki log record
+        app: frontend # static value
+        kind:         # value will be taken from log items. exception, log, measurement, etc
+      logs_send_timeout: 5000
+      sourcemaps:
+        download: true # will download source file, extract source map location,
+        # download source map and use it to transform stack trace locations
+```
+
+## Install Grafana Javascript Agent
+
+```bash
+#npm
+npm i -S @grafana/agent-web
+
+#yarn
+yarn add @grafana/agent-web
+```
+
+## Initialize Grafana Javascript Agent
+
+### Basic
+
+Will set up agent to automatically collect uncaught errors
+and [web vitals](https://github.com/GoogleChrome/web-vitals) measurements. No tracing, low footprint.
+
+```javascript
+import { initializeAgent } from '@grafana/agent-web';
+
+const agent = initializeAgent({
+  url: 'https://collector-host:12345/collect',
+  apiKey: 'secret',
+  app: {
+    name: 'frontend',
+    version: '1.0.0',
+  },
+});
+```
```

Comment:
By "initialize" - do you mean run?

Reply:
What is basic? is there also advanced? what other options, if any?

Reply:
I believe "initialize" is the correct technical term here. No new processes will be started or anything of that nature. Rather, library will be configured and hook into some browser mechanisms.

Basic = simple configuration, mostly using default implicit options
Advanced = customizing "advanced" options like transports and instrumentations, which shouldn't be necessary for most users. I can try to add a section about it

Reply:
How about something like this then: This basic configuration sets up the Grafana Javascript Agent to automatically collect uncaught errors. (undetected?). No tracing is involved, so this means a low footprint. (again not sure what you mean by low footprint...but if it's obvious then fine...)

Reply:
Updated worning. "uncaught" as a specific technical term. In JS errors are "caught" using the `catch {}` block. So if that didn't happen, error is considered "uncaught" :)